### PR TITLE
update ui field and classnames

### PR DESCRIPTION
**Pull request by rianfowler**
## Description
Addresses [this](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/12670#issuecomment-423239754) comment

## Testing done
ran locally and tabbed through fields verifying that the informational element is in the tab order

## Acceptance criteria
- [x] element should render without a wrapping legend element

## Definition of done
~- [ ] Events are logged appropriately~
~- [ ] Documentation has been updated, if applicable~
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
~- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs~

### CODE REVIEW

File: src/applications/edu-benefits/0993/config/form.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ const formConfig = {
               }
             },
             'view:optOutMessage': {
-              'ui:title': tabIndexedTitle('By clicking the Continue button, you’re asking VA to not share your education benefits information.')
+              'ui:field': () => tabIndexedTitle('By clicking the Continue button, you’re asking VA to not share your education benefits information.')
```
Comment (rianfowler) [inline src/applications/edu-benefits/0993/config/form.js]: `ui:title` includes a `<legend>` which was interfering with one of the accessibility tests.

**Comment by rianfowler** [comment]
@1Copenut LMK what you think of the new markup.

File: src/applications/edu-benefits/0993/config/form.js (hunk 1 above)
Comment (cehsu) [reply src/applications/edu-benefits/0993/config/form.js]: Good to know, we will probably need to apply this pattern/fix, in many other places.

**Review by cehsu — APPROVED** [review]
LGTM.

**Comment by 1Copenut** [comment]
@rianfowler I'll take it for a screen reader test drive yet this afternoon. Thank you for the quick turnaround.

**Comment by 1Copenut** [comment]
@rianfowler Retested in IE11 + JAWS and now reading as expected. Will test other screen reader combos but this was the big one. Thank you for re-working the output.

**Pull request merged**